src/components/Tabs/Tabs.scss

```diff
@@ -6,6 +6,18 @@ $item-vertical-padding: $item-min-height / 2;
 $underline-height: border-width(thicker);
 $focus-state-box-shadow-color: rgba(0, 0, 0, 0.8);
 
+@mixin reset-before-pseudo-content {
+  &::before {
+    content: none;
+  }
+}
+
+@mixin focus-ring-override {
```

Comment:
seeing these mixins playing together is like a beautiful symphony 🤩

Reply:
ALL thanks to @beefchimi showing me all the cool tricks this morning 🧙🏽‍♂️